src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp

```diff
@@ -34,6 +34,7 @@ SCENARIO( "Integration test" ){
   SECTION( "Cobalt-58" ){
     auto Co58 = resonances("Co-58");
 
+    auto start = std::chrono::high_resolution_clock::now();
```

Comment:
you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csmlbw`, you should move this to (what is now) line 52